.buildkite/dagster-buildkite/dagster_buildkite/steps/integration.py

```diff
@@ -30,6 +30,39 @@ def integration_suite_extra_cmds_fn(version):
     ]
 
 
+def backcompat_suite_extra_cmds_fn(release_mapping):
+    dagit_version = release_mapping["dagit"]
+    user_code_version = release_mapping["user_code"]
+
+    if dagit_version == "current_branch":
```

Comment:
maybe current_branch should be a const to make it clear that it has a semantic meaning?